src/clients/python/python_bindings.zig

```diff
@@ -0,0 +1,497 @@
+const std = @import("std");
+const vsr = @import("vsr");
+const tb_client = vsr.tb_client;
+const assert = std.debug.assert;
+
+const constants = vsr.constants;
+const IO = vsr.io.IO;
+
+const Storage = vsr.storage.StorageType(IO);
+const StateMachine = vsr.state_machine.StateMachineType(Storage, constants.state_machine_config);
+const tb = vsr.tigerbeetle;
+
+/// VSR type mappings: these will always be the same regardless of state machine.
+const mappings_vsr = .{
+    .{ tb_client.tb_operation_t, "Operation" },
+    .{ tb_client.tb_packet_status_t, "PacketStatus" },
+    .{ tb_client.tb_packet_t, "Packet" },
+    .{ tb_client.tb_client_t, "Client" },
+    .{ tb_client.tb_status_t, "Status" },
+};
+
+/// State machine specific mappings: in future, these should be pulled automatically from the state
+/// machine.
+const mappings_state_machine = .{
+    .{ tb.AccountFlags, "AccountFlags" },
+    .{ tb.TransferFlags, "TransferFlags" },
+    .{ tb.AccountFilterFlags, "AccountFilterFlags" },
+    .{ tb.QueryFilterFlags, "QueryFilterFlags" },
+    .{ tb.Account, "Account" },
+    .{ tb.Transfer, "Transfer" },
+    .{ tb.CreateAccountResult, "CreateAccountResult" },
+    .{ tb.CreateTransferResult, "CreateTransferResult" },
+    .{ tb.CreateAccountsResult, "CreateAccountsResult" },
+    .{ tb.CreateTransfersResult, "CreateTransfersResult" },
+    .{ tb.AccountFilter, "AccountFilter" },
+    .{ tb.AccountBalance, "AccountBalance" },
+    .{ tb.QueryFilter, "QueryFilter" },
+};
+
+const mappings_all = mappings_vsr ++ mappings_state_machine;
+
+const Buffer = struct {
+    inner: std.ArrayList(u8),
+
+    pub fn init(allocator: std.mem.Allocator) Buffer {
+        return .{
+            .inner = std.ArrayList(u8).init(allocator),
+        };
+    }
+
+    pub fn print(self: *Buffer, comptime format: []const u8, args: anytype) void {
+        self.inner.writer().print(format, args) catch unreachable;
+    }
+};
+
+fn mapping_name_from_type(mappings: anytype, Type: type) ?[]const u8 {
+    inline for (mappings) |mapping| {
+        const ZigType = mapping[0];
+        const name = mapping[1];
```

Comment:
```suggestion
        const ZigType, const name = mapping;
```

Reply:
and probably could be comptime for rather than inline? `comptime` is better as it is more specific (every comptime for can be an inline one, but not every inline for can be replaced with comptime for)